crates/static-analysis-kernel/src/analysis/analyze.rs

```diff
@@ -537,6 +540,65 @@ def foo(arg1):
         );
     }
 
+    #[test]
+    fn test_go_file_context() {
+        let code = r#"
+import (
+    "math/rand"
+    crand1 "crypto/rand"
+    crand2 "crypto/rand"
+)
+
+func main () {
+
+}
+        "#;
+
+        let query = r#"(function_declaration) @func"#;
+
+        let rule_code = r#"
+function visit(node, filename, code) {
+    const n = node.captures["func"];
+    console.log(node.context.packages);
+    if(node.context.packages.includes("math/rand")) {
```

Comment:
it's a documentation problem/issue and I do not think we should add too many things under `context`. Otherwise, we will end up with
`context.<language-name>`
`context.<library-name>`
etc, etc.